extensions/pyRevitTools.extension/checks/radar_check.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
+
+import math
+# pyRevit
+from pyrevit import revit, DB, DOCS, HOST_APP
+from pyrevit import script
+from pyrevit.preflight import PreflightTestCase
+
+
+doc = DOCS.doc  # Current Document
+INTERNAL_ORIGIN = (0, 0, 0)
+EXTENT_DISTANCE = 52800  # Linear Feet
+BAD_STRG = '### :thumbs_down_medium_skin_tone: ............'
+GOOD_STRG = '### :OK_hand_medium_skin_tone: ............'
+WARN_STRG = '### :warning: ............'
+CRITERIA_STRG = '10 Mi (16KM) away from the Internal Origin.'
+
+# ___________________________________________________3D to Bounding Box Analysis
```

Comment:
what's whit this comment style?

use docstrings for documentation purposes, and `# region <name> ... # endregion` to mark regions of related code.